infra/main.bicep

```diff
@@ -0,0 +1,152 @@
+var rg = resourceGroup()
+var location = resourceGroup().location
+
+@description('Suffix to create uniqute resource names. 4-6 symbols. Default is random 6 symbols.')
+@minLength(4)
+@maxLength(6)
+param suffix string = substring(newGuid(), 0, 6)
+
+// //az deployment sub create -f main.bicep --location=$Location --parameters location=$Location -c
+// targetScope = 'subscription'
+// param location string
+// resource rg 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+//   name: 'km-rg-${suffix}'
+//   location: location
+// }
+
+module managedidentity 'teamplates/identity.bicep' = {
```

Comment:
could you please add a comment for all modules, with a brief explanation, e.g. what the resource is, how it is used, and perhaps a link to its documentation?